src/components/NcButton/NcButton.vue

```diff
@@ -509,13 +509,10 @@ export default {
 	}
 
 	&__icon {
-		height: $clickable-area;
-		width: $clickable-area;
-		min-height: $clickable-area;
-		min-width: $clickable-area;
 		display: flex;
 		justify-content: center;
 		align-items: center;
+		margin: auto 8px auto 12px;
```

Comment:
Where are those random values coming from?

Reply:
Random to how it looks good :see_no_evil:. Left one is a bit higher so it looks properly with the rounded pill.

Reply:
These brakes the icon-only buttons. It also seems that they accomplish the same as sizing the icon container. What's the reasoning here?

Reply:
Won't work like this and is very subjective. I would leave it as it is, since it's not related to the intention of this PR of fixing the button height.

Reply:
Reason was, that the height was fixed twice, so it was necessary with the border thing and imo this would have been a bit cleaner. Anyways. Reverted it as of Raimunds comment. :wink:

Reply:
Wow we spammed this poor comment xD

Reply:
Yes, wth guys :grin: 

In any case, no random values, base the sizing on `clickable-area` pls :)